Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -392,8 +401,10 @@ void ctkDICOMBrowserPrivate::init()
   q->connect(importDirectoryModeComboBox, SIGNAL(currentIndexChanged(int)),
           q, SLOT(onImportDirectoryComboBoxCurrentIndexChanged(int)));
 
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.